codalab/worker/docker_utils.py

```diff
@@ -280,10 +281,16 @@ def get_image_size_without_pulling(image_spec):
     """
     logger.info("Downloading tag information for {}".format(image_spec))
     image_name, image_tag = image_spec.split(":")
-    # Example URL: https://hub.docker.com/v2/repositories/codalab/default-cpu/tags
+    # Example URL:
+    # 1. image with namespace: https://hub.docker.com/v2/repositories/<namespace>/<image_name>/tags
+    #       e.g. https://hub.docker.com/v2/repositories/codalab/default-cpu/tags
+    # 2. image without namespace: https://hub.docker.com/v2/repositories/library/<image_name>/tags
+    #       e.g. https://hub.docker.com/v2/repositories/library/ubuntu/tags/
     # Note that since docker-py doesn't report the accurate compressed image size, e.g. the size reported
     # from the RegistryData object, we then switch to use Docker Registry HTTP API V2
-    request = DOCKER_REGISTRY_HTTP_API_V2_PREFIX + image_name + '/tags'
+    # URI prefix of an image without namespace will be adjusted to https://hub.docker.com/v2/repositories/library
+    URI_prefix_adjusted = URI_prefix + '/library/' if '/' not in image_name else URI_prefix
```

Comment:
similarly can we keep this variable name all lowercase since this makes syntax checkers think this is a constant.

Reply:
Sure. Good point!